charts/ziti-router/templates/configmap.yaml

```diff
@@ -65,7 +65,9 @@ data:
             mode: {{ .Values.tunnel.mode }}
           {{- end }}
           {{- if eq .Values.tunnel.mode "tproxy" }}
+            {{- if .Values.tunnel.resolver }}
```

Comment:
@dariuszSki If not specified, then the default `resolver` is used, correct?

Reply:
In tproxy mode,you need to set the resolver. I never tested without setting one.

Reply:
I don't think there is a default one if you don't configure one with tproxy

Reply:
I confirmed the default resolver is used if `resolver` property is absent, so this is the correct implementation for the chart, to only set that property with the input value is defined.